cypress/support/step-definitions/dialog-steps.js

```diff
@@ -1,11 +1,27 @@
-import { alertDialogPreview as dialogPreview, dialogStickyFormFooter, dialogStickyFormFooterButton } from '../../locators/dialog/index';
-import { closeIconButton, backgroundUILocator, storyRoot } from '../../locators/index';
-import { positionOfElement } from '../helper';
+import { 
+  alertDialogPreviewNoIFrame as dialogPreview,
```

Comment:
Why you can use only one a proper locator ?

Reply:
to make alert locator used in dialog and as dialog :)